src/graph.rs

```diff
@@ -918,6 +1016,7 @@ pub struct PushScopedSymbolNode {
     pub scope: Handle<Node>,
     /// Whether this node represents a reference in the source language.
     pub is_reference: bool,
+    _phantom: (),
```

Comment:
Why is this necessary?

Reply:
It means that you can't instantiate this type from outside this file, you have to go through `StackGraph::add_push_scoped_symbol_node` to create them.  All of the other node types have some other private field that prevents this, but this type didn't, so I needed to add a private zero-sized field to get this behavior.